cpp/src/arrow/json/test_common.h

```diff
@@ -99,7 +99,7 @@ struct GenerateImpl {
   template <typename T>
   enable_if_base_binary<T, Status> Visit(const T&) {
     auto size = std::poisson_distribution<>{4}(e);
-    std::uniform_int_distribution<uint16_t> gen_char(32, 127);  // FIXME generate UTF8
+    std::uniform_int_distribution<uint16_t> gen_char(32, 126);  // FIXME generate UTF8
```

Comment:
I assume that ASCII DEL wasn't intended to be included in the test input...

Reply:
Probably. Is it forbidden in JSON?

Reply:
They're legal. It just seemed that the author meant to exclude non-printable characters entirely (although [x00-x1f need to be escaped but x7f doesn't](https://www.rfc-editor.org/rfc/rfc8259#section-7), apparently).